File: var/www/cgi/env.rb
#!/usr/bin/ruby

puts("<html>")

puts("<head>")
puts("<h2>Environment with RUBY</h2><br>")
puts("</head>")

puts("<body>")
# Loop through all environment variables and print them
ENV.each do |key, value|
  puts "#{key}=#{value}<br>"
end
puts("</body>")

puts("</html>")

File: var/www/cgi/env.py
#!/usr/bin/python3

import os
import cgi

print("<html>")

print("<head>")
print("<h2>Environment with PYTHON</h2><br>")
print("</head>")

print("<body>")
for param in os.environ.keys():
    print("<b>%20s</b>: %s<br>" % (param, os.environ[param]))
print("</body>")

print("</html>")